### ISSUE TITLE
How I can use trained Regression model on new input feature

### ISSUE THREAD

**Issue by dheeraj-2000** [shankarpandala/lazypredict#167]
Hii
I'm using LazyPredict to get performance of Regression models on a dataset (with LazyRegressor), and finally I am getting the performance table and prediction values from this

```
regr=LazyRegressor(verbose=0,predictions=True)   
models_r,predictions_r=regr.fit(X_train, X_test, Y_train, Y_test)
```

But now I want to know the prediction values on new input features. 

So, how I can use this **model_r** to get the prediction table? or is there other way to get it ?

**Comment by shankarpandala** [comment]
It is not possible to do that with current version.
I am planning to release it in the next version.

If you want to give a try, clone the dev branch of this repository and try. It creates two folders, one of which contains piclke files of all the models.

I haven't tested it yet.

**Comment by dheeraj-2000** [comment]
@shankarpandala Okay sure! I'll also try it.

Thank you.

**Comment by shyamcody** [comment]
@dheeraj-2000 we have added a feature provide_models ( not released yet) in supervised.py. This will let you now download the models trained. I guess that should solve this issue.

**Comment by dheeraj-2000** [comment]
@shankarpandala that's great ✌️
I'll check it as soon as possible.

Thank you

**Comment by brendalf** [comment]
We've just released the lazypredict v0.2.8 with the `provide_models` that @shyamcody annunced.
@dheeraj-2000  can you try it? I'm closing this issue for now.
